### ISSUE TITLE
Automatic task creation does not assign tasks to user with user role Surveillance Officer

### ISSUE THREAD

**Issue by KristinaBorch** [hzi-braunschweig/SORMAS-Project#9047]
### Bug Description

When new cases are imported and the automatic task creation is activated, the tasks are automatically assigned to the person performing the import and not to the user with the role Surveillance Officer.  

### Steps to Reproduce

The Bug can be reproduced with the following user role combinations:

- Admin + Surveillance Supervisor

- Admin + Surveillance Officer

- Admin + Admin Surveillance Supervisor

### Expected Behavior

The automatic task creation assignes the task to the user with the role Surveillance Supervisor.

### Additional Information

[sormas_import_case_template_V1.70.csv](https://github.com/hzi-braunschweig/SORMAS-Project/files/8609988/sormas_import_case_template_V1.70.csv)
### System Details

* SORMAS version: test1402

**Comment by MateStrysewske** [comment]
@KristinaBorch That's not a bug. All three user roles (Surveillance Supervisor, Surveillance Officer and Admin Surveillance Supervisor) are allowed to be set as responsible users of cases, and the responsible user of a case is always the user who's assigned to tasks related to that case. If something is supposed to be changed about this behaviour, please open a change request with detailed instructions concerning the order in which case tasks should be assigned to users.

**Comment by herrling-LRA-KYF** [comment]
Its also for us difficult to handle, because we have a lot of users with the role "Surveillance Supervisor". before the last update, we used the "TASK_GENERATION_CASE_ SURVEILLANCE  ", which only create a task for users with the role "Surveillance Officer". In our case, only survNet had this role.
And we created an own RPA via the rest API to manage a lot of stuff and fill out a lot of fiels automatically.
